core/src/main/java/org/apache/carbondata/core/metadata/schema/table/TableSchemaBuilder.java

```diff
@@ -117,7 +116,7 @@ public TableSchema build() {
       property.put(CarbonCommonConstants.TABLE_BLOCKSIZE, String.valueOf(blockSize));
     }
     if (blockletSize > 0) {
-      property.put(CarbonV3DataFormatConstants.BLOCKLET_SIZE_IN_MB, String.valueOf(blockletSize));
```

Comment:
Why not use the old variable directly?

Reply:
CarbonV3DataFormatConstants.BLOCKLET_SIZE_IN_MB is  "carbon.blockletgroup.size.in.mb", this string is system property, but a normal table property is without dot, like `TABLE_BLOCKSIZE = "table_blocksize";
So I added a `TABLE_BLOCKLET_SIZE = "table_blocklet_size"`